packages/webpack-stats-report/src/createReport.ts

```diff
@@ -23,7 +23,7 @@ const getPercentage = (reportAssetData: ReportAssetData) => {
   const percentage = (100 / refSize) * Math.abs(reportAssetData.diff);
 
   if (reportAssetData.isIncrease) {
-    return `+${percentage.toFixed(2)}%`;
+    return percentage > 20 ? `<b>+${percentage.toFixed(2)}%</b>` : `+${percentage.toFixed(2)}%`;
```

Comment:
Same here, we decided to start with 5% rather 20%

Reply:
Should be good now